src/components/backgroundBlur/BackgroundBlur.jsx

```diff
@@ -0,0 +1,8 @@
+import "./BackgroundBlur.css"
+
+export default function backgroundBlur({handleSummaryDetailClose}) {
```

Comment:
Function names should start with Capital Letters.

Reply:
Resolved with commit [997c989](https://github.com/chingu-voyages/v46-tier2-team-20/pull/51/commits/997c98971aaf211d9e518c3dc124781e917bcad0), below